src/collections/_documentation/platforms/android/index.md

```diff
@@ -300,22 +292,60 @@ sentry-cli upload-proguard \
+- Filter events and issues by the release version directly in the tag search on the Issues and Events pages
+- Check what new issues were introduced with the new release
+
+With the releases and a [GitHub]({%- link _documentation/workflow/integrations/global-integrations.md -%}#github)/[GitLab]({%- link _documentation/workflow/integrations/global-integrations.md -%}#gitlab) integration, you can:
+
+- Determine the issues and regressions introduced in a new release
+- Receive suggestions about which commit caused an issue and who is likely responsible
 - Resolve issues by including the issue number in your commit message
-- Receive email notifications when your code gets deployed
+- Receive email notifications when your code deploys
+
+Even though releases are automatically created as events come in, to take advantage of the *suspected commits* feature, you need to create the release in Sentry as part of your build process.
 
-After configuring your SDK, setting up releases is a 2-step process (recommended for the use of suspect commits):
+Sentry offers a command-line tool to aid with this task. After configuring your SDK, setting up releases is a 2-step process (recommended for the use of suspect commits):
 
 1. [Create Release and Associate Commits](https://docs.sentry.io/workflow/releases/#create-release)
 2. [Tell Sentry When You Deploy a Release](https://docs.sentry.io/workflow/releases/#create-deploy)
 
 For more information, see [Releases Are Better With Commits](https://blog.sentry.io/2017/05/01/release-commits.html).
 
+### Release Version Format
+
+The default format of the release version that is sent with each event from the SDK is:
+
+```
+packageName@versionName+versionCode
+```
+
+Please note that if you're using multiple flavors in your application, the release version will be different for each flavor, and different release objects will be created in-app for each flavor.
+
+If you want to change the release name, you can do it in the `AndroidManifest.xml` or directly in the code. 
+
+The release version can be any random string, but we recommend using a similar format to the default. The default involves having the text identifier of your app connected with the version string using "@" and the last optional suffix dedicated to build or an additional identifier. With this format, the Sentry UI will display a more comprehensive release name. For example, company.demo.app@1.1.1 instead of 1.1.0.
+
+To change the release version in the `AndroidManifest.xml`:
+
+```xml
+<meta-data android:name="io.sentry.release" android:value="io.example@1.1.0" />
+```
+
+Or, you can set the release version in your code during the manual initialization of the SDK as described in [Manual Initialization](#manual-initialization).
+
+```java
+SentryAndroid.init(this, options -> {
+    options.setRelease("io.sentry@1.1.0");
```

Comment:
I'd set `io.example@1.1.0` so it's understood that it's the same as the `xml` conf.